### ISSUE TITLE
"Galaxy Cluster" parameter for Celestia's DSC file

### ISSUE THREAD

**Issue by LafuenteAstronomy** [CelestiaProject/Celestia#735]
Hello there
Something just came to my mind while seeing some pictures of a galaxy cluster from the Discord Server, and I suggested it there a few times: Is it possible to add the "Galaxy Cluster" type to Celestia' list of DSC types? In that sense that we can have an identifier and characteristics for a Galaxy Cluster with individual galaxies in as much the same way as OpenClusters and GlobularClusters can have individual members as rendered by Celestia? 

If ever this is implemented, I suggest that the Local Group be the very first object of that type to be created

Thanks in advance

**Comment by SevenSpheres** [comment]
Good idea, it would probably work like open clusters. Now where to find a catalog of galaxy clusters...

**Comment by LafuenteAstronomy** [comment]
> Good idea, it would probably work like open clusters. Now where to find a catalog of galaxy clusters...

The Abell Catalog will be a good start

**Comment by LafuenteAstronomy** [comment]
https://en.wikipedia.org/wiki/Abell_catalogue
